### ISSUE TITLE
updating the system install

### ISSUE THREAD

**Issue by Anniepoo** [fnogatz/swivm#5]
It's a bit unclear to me whether installing from sources will 'break' swivm.

Here's my scenario -
We have a breaking bug, big priority for us. Jan pushes a fix. I'd like to start using the fix immediately, but don't want to trash my swivm install.

How should I proceed?

**Comment by Anniepoo** [comment]
since this is a documentation issue anyway - 
installing does same as 
swivm deactivate
and lets you use the one installed from source

You can then use 
swivm use 7.4.1 or whatever to go to another version.

I'm not sure what 'system' version is, had no luck with that

**Comment by fnogatz** [comment]
The `system` version is this one not installed by swivm. If you have installed an SWI version by, e.g., `apt-get install swi-prolog` or system-wide self-compiled, this will be the `system` version.

**Comment by fnogatz** [comment]
I am not sure if I understand correctly: Do you want to install the latest development version of SWI from source using `swivm`, even if it is not yet released under a specific version number?

Or do you want to know what's the best way to use a self-compiled SWI version in parallel with `swivm`?

**Comment by Anniepoo** [comment]
I eventually figured this out. It's not clear from the readme what the 'system' version is.

I'd suggest adding "The system version is this one not installed by swivm. If you have installed an SWI version by, e.g., apt-get install swi-prolog or system-wide self-compiled, this will be the system version."

to the readme in some form.

**Comment by fnogatz** [comment]
Thank you for the clarification and the suggestion to improve the readme. I added your suggested text to the readme.

I will close here. If you have further improvement ideas, please feel free to open a new issue :)
